### PR TITLE
Add Home Page

### PR DESCRIPTION
**Pull request by ashleyplusplus**
Added homepage. Based it off of the Figma w/ minor spacing adjustments to set styling baseline and reflect what we spoke about regarding color scheme with Dr. Anderson. However, this can certainly be adjusted in the future. Also, purposely kept previous background and font color commented out in case we want to adjust back to those.

See screenshots below to view what changes look like/how they compare to Figma without checking out branch.

Edit: team decided to make background slightly lighter.

Figma link: https://www.figma.com/file/GBZ4l4aFqGMEgepbghjmmz/Mid-Lesson-Page-Concept-Design?node-id=37%3A41&mode=dev

Figma:
image

**UPDATED** page created locally in HTML/CSS:
image

Page created locally in HTML/CSS **(OLD)**:
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,13 +4,31 @@
 <head>
     <title>Home</title>
     <link rel="stylesheet" href="index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Tourney:ital,wght@0,100..900;1,100..900&display=swap"
+        rel="stylesheet">
```
Comment (ashleyplusplus) [inline index.html:10]: pulls in Tourney font (Google font free to use)

**Comment by y1c1** [comment]
Looks good to me!

**Comment by BethJ-20** [comment]
It looks great! Maybe we change the background color to make it more readable but I don't foresee that being an issue

**Review by Jsharp517 — APPROVED** [review]
This looks great! Good work.

**Review by BethJ-20 — APPROVED** [review]

**Review by y1c1 — APPROVED** [review]

**Pull request merged**